### PR TITLE
[Refactoring] new main page tests

### PR DESCRIPTION
**Pull request by mmajerni**

### CODE REVIEW

File: src/test/resources/features/xMainPageTests.feature [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    And save API as "json" and close editor
+    And import API "tmp/download/openapi-spec.json"
+    Then check that API name is "MyApi"
+
+  @setAPIversion
+  Scenario: set API version
+    When set API version to "2.20a"
+    And save API as "json" and close editor
+    And import API "tmp/download/openapi-spec.json"
+    And check that API version is "2.20a"
```
Comment (mcada) [inline src/test/resources/features/xMainPageTests.feature]: Should be then

File: src/test/java/apicurito/tests/steps/verification/MainPageVerifications.java [hunk 2]
```diff
@@ -54,28 +55,29 @@ public void checkThatAPIProduce(String expectedProduce) {       //NEW
...
+            assertThat(name).as("Checking API contatct name:").isEqualTo(dataRow.get(0));
+
             String email = MainPageUtils.getMainPageRoot().$("contact-section").$(By.className("email")).getText();
-            String url = MainPageUtils.getMainPageRoot().$("contact-section").$(By.className("url")).getText();
+            assertThat(email).as("Checking API contatct email:").isEqualTo(dataRow.get(1));
 
-            CollectorHelper.getCollector().assertThat(name).as("Checking API contatct name:").isEqualTo(dataRow.get(0));
-            CollectorHelper.getCollector().assertThat(email).as("Checking API contatct email:").isEqualTo(dataRow.get(1));
-            CollectorHelper.getCollector().assertThat(url).as("Checking API contatct url:").isEqualTo(dataRow.get(2));
+            String url = MainPageUtils.getMainPageRoot().$("contact-section").$(By.className("url")).getText();
```
Comment (mcada) [inline src/test/java/apicurito/tests/steps/verification/MainPageVerifications.java]: Do you plan to add inside class with all element selectors as we discussed?

File: src/test/resources/features/xMainPageTests.feature (hunk 1 above)
Comment (mmajerni) [reply src/test/resources/features/xMainPageTests.feature]: updated

File: src/test/java/apicurito/tests/steps/verification/MainPageVerifications.java (hunk 2 above)
Comment (mmajerni) [reply src/test/java/apicurito/tests/steps/verification/MainPageVerifications.java]: yes, I have already created a jira task.

Comment (mcada) [reply src/test/java/apicurito/tests/steps/verification/MainPageVerifications.java]: ok

**Pull request merged**